src/java/org/apache/cassandra/streaming/async/StreamingMultiplexedChannel.java

```diff
@@ -90,8 +88,8 @@
 {
     private static final Logger logger = LoggerFactory.getLogger(StreamingMultiplexedChannel.class);
 
-    private static final int DEFAULT_MAX_PARALLEL_TRANSFERS = getAvailableProcessors();
-    private static final int MAX_PARALLEL_TRANSFERS = parseInt(getProperty(PROPERTY_PREFIX + "streaming.session.parallelTransfers", Integer.toString(DEFAULT_MAX_PARALLEL_TRANSFERS)));
+    public static final int DEFAULT_MAX_PARALLEL_TRANSFERS = getAvailableProcessors();
```

Comment:
why does this have to be `public`?

Reply:
My bad. Previously, I've tried to move such defaults to another class but returned it.
Fixed.